src/tests/routers/generic/test_router_post.py

```diff
@@ -13,20 +15,22 @@ def test_unicode(client_test_resource: TestClient, title: str, auto_publish: Non
     with logged_in_user():
         response = client_test_resource.post(
             "/test_resources/v0",
-            json={"title": title, "platform": "example", "platform_resource_identifier": "1"},
+            json={"title": title},
             headers={"Authorization": "Fake token"},
         )
     assert response.status_code == 200, response.json()
-    identifier = response.json()['identifier']
+    assert "identifier" in response.json()
+    identifier = response.json()["identifier"]
 
     response = client_test_resource.get(f"/test_resources/v0/{identifier}")
     assert response.status_code == 200, response.json()
     response_json = response.json()
     assert response_json["title"] == title
+    assert response_json["platform"] == PlatformName.aiod
 
 
 def test_missing_value(client_test_resource: TestClient):
-    body = {"platform": "example", "platform_resource_identifier": "1"}
+    body = {} # type: ignore
     with logged_in_user():
         response = client_test_resource.post(
             "/test_resources/v0", json=body, headers={"Authorization": "Fake token"}
```

Comment:
```suggestion
    with logged_in_user():
        response = client_test_resource.post(
            "/test_resources/v0", json=dict(), headers={"Authorization": "Fake token"}
```
This probably also works. Otherwise,
```
body: dict[str, str] = {}
```
should work.

Reply:
The problem here is that mypy wants to know the type of the keys and values in the dictionary, but it does not have context to infer it (anymore). So it just needs some clarification on the type of the variable. Either providing this explicitly, or never assigning it a variable should both work.